configuration/doc/ONELINECRBOT.md

```diff
@@ -0,0 +1,108 @@
+The basic functionality of the one line cr bot is described in the [doc/README.md](https://github.com/awslabs/one-line-scan/blob/master/configuration/doc/README.md)
+file. Examples to use the file are given in the [doc/EXAMPLES.md](https://github.com/awslabs/one-line-scan/blob/master/configuration/doc/EXAMPLES.md) file. This file
+highlights using one-line-cr-bot.sh as part of CI systems.
+
+# Workflows
+
+There are workflows that allow to automate analysis, and also block. The below
+YAML file can be used as an example to start implementing code analysis for your
+C and C++ projects using github workflows.
+
+## Github Workflows
+
+Github workflows are activated by placing a YAML file in the .github/workflows
+directory. By placing the below file into that directory, i.e. storing the full
+below content in the file .github/workflows/one-line-cr-bot.yml any future pull
+request will receive code analysis.
+
+The file has sections labeled "[ACTION REQUIRED]" which likely need to be
+adapted to meet the configuration of your project.
+
+In case the analysis creates many false positive alarms for your project,
+consider to pass the parameter '-y' to the one-line-cr-bot.sh call. This will
+still run the analysis, but will not fail in case of errors. When doing so,
+spotting new defects becomes a more manual process, but will not block your pull
+requests unnecessarily.
+
+    # Author: Norbert Manthey <nmanthey@amazon.de>
+    #
+    # This workflow will present introduced defects of a pull request to a given
+    # branch of a package.
```

Comment:
How about: "This workflow will report defects newly introduced by a pull request as compared to the base of the branch."